install/kubernetes/helm/istio/values.yaml

```diff
@@ -157,7 +157,8 @@ ingressgateway:
     type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
     ports:
       ## You can add custom gateway ports
-    - port: 80
+    - port: 15001
+      targetPort: 80
```

Comment:
? this looks like a one off hack. why change the gateway to use a new port?

Reply:
@rshriram  This port is service port but not container port, the container port is `targetPort`, it is still 80.

You can check my case here https://github.com/istio/istio/issues/6000#issue-328886754 for detail.

Reply:
Yes, but we have do it this way, otherwise, the iptables rules may direct the request to `http://ingress-gateway-ip:80` to another services's 80 port.

How about define the default value of `port` as 80 as well here so that customer can customize it based on demand.

Reply:
@rshriram @gyliu513 defining the default value of service port as 80 and exposing service port in `values.yaml` is a good option, like this:
```
ports:
    ## You can add custom gateway ports
    port: 15001
    targetPort: 80
    ...
```